apps/veil/src/pages/tournament/ui/vote-dialog/vote-dialog-asset.tsx

```diff
@@ -1,48 +1,70 @@
 import cn from 'clsx';
 import { AssetIcon } from '@penumbra-zone/ui/AssetIcon';
+import { round } from '@penumbra-zone/types/round';
 import { Dialog } from '@penumbra-zone/ui/Dialog';
 import { Text } from '@penumbra-zone/ui/Text';
 import type { MappedGauge } from '../../server/previous-epochs';
 
-const VOTING_THRESHOLD = 0.05;
+export const VOTING_THRESHOLD = 0.05;
 
 export interface VoteDialogAssetProps {
   asset: MappedGauge;
   onSelect: VoidFunction;
 }
 
-export const VoteDialogAsset = ({ asset, onSelect }: VoteDialogAssetProps) => {
+export const getVoteAssetComponents = (asset: MappedGauge) => {
```

Comment:
I think it's probably more idiomatic to split this into two components instead

Reply:
good point, refactored